cafebabel/static/js/script.js

```diff
@@ -26,6 +26,36 @@ Array.from(document.querySelectorAll('a')).forEach(a => {
     a.setAttribute('target', '_blank')
 })
 
+/* smooth scroll to anchor https://stackoverflow.com/a/17733311/6481285 */
+function animate(elem, style, from, to, time) {
+  if (!elem) return
+  const start = new Date().getTime()
+  const timer = setInterval(() => {
+    const step = Math.min(1, (new Date().getTime() - start) / time)
+    elem[style] = from + step * (to - from)
+    if (step === 1) clearInterval(timer)
+  })
+  elem[style] = from
+}
+const subscribeLink = document.querySelector(
+  '#social-networks .icon-newsletter'
+)
+subscribeLink.addEventListener('click', () => {
+  const target = document.querySelector('#static-pages-links')
```

Comment:
It should be retrieved from the `event` like `event.target.href`

Reply:
This smooth scroll doesn't work for me.
It just goes to the anchor as normal.
I tried prepending with `event.preventDefault()` to cut off the default behavior (that I suppose we need to turn off) and it wouldn't move at all anymore.

Reply:
ah oui, on firefox, it doesn't work... desperate..

Reply:
Fixed here @vinyll ebc33fc 😅